applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                        by inspecting the direction of their outer unit normals (which point either parallel or antiparallel)
+                        along respective base vector direction of the reference frame.
+                        
+                        If any of these assumptions is not met, the user is required to explicitly state this.
+                    </doc>
+                    <enumeration>
+                        <item value="front_top_left"/>
+                        <item value="front_top_right"/>
+                        <item value="front_bottom_right"/>
+                        <item value="front_bottom_left"/>
+                        <item value="back_top_left"/>
+                        <item value="back_top_right"/>
+                        <item value="back_bottom_right"/>
+                        <item value="back_bottom_left"/>
+                    </enumeration>
+                </field>
+                <field name="x_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing x-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="y_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing y-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="z_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing z-axis base vector of the
+                        detector_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+            </group>
+        </group>
+        <group name="measurement" type="NXobject" optional="true">
```

Comment:
NXobject?

Reply:
Refactored to tiny base class NXem_measurement
The use of new such tiny base classes whose purpose is mainly that of an NXbag with validation possible has been kept to a minimum see also the following comment https://github.com/FAIRmat-NFDI/nexus_definitions/commit/9a1f44cc07617b2646ac149fc4431333614f9d5e